ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -316,18 +316,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn
-- as: azure-ipi-private-p2-f7
-  cron: 29 22 4,12,19,26 * *
+- as: azure-ipi-private-workers-marketimage-p2-f14
```

Comment:
For this job, I suggest to keep the same frequency as before on `f7`

Reply:
Done